client/src/components/app.jsx

```diff
@@ -168,7 +171,36 @@ class App extends React.Component {
     }
   }
 
+  updateUser = (user) => {
+    this.setState(
+      {user: user}
+    )
+  }
+
+  checkLoginState = () => {
+    axios.get('/status')
+    .then((response) => {
+      this.setState({
```

Comment:
Interesting, why did you choose to use stateful components? Try to verbalize an answer to this. If I were to look at the rest of the app, would other components be stateful? No need to update your code btw! I'm just asking you to verbalize why you made this choice